restapi/src/test/java/io/stargate/web/docsapi/service/QueryExecutorTest.java

```diff
@@ -50,6 +50,7 @@
 import org.junit.jupiter.params.provider.CsvSource;
 
 class QueryExecutorTest extends AbstractDataStoreTest {
+  private DocsApiConfiguration config = DocsApiConfiguration.DEFAULT;
```

Comment:
For proper unit tests, we should use the mocked version, mock depending methods (I think this test has none) and then verify component is using values the mocked values..